README.md

```diff
@@ -27,6 +27,7 @@ OPTIONS:
                           Set the print format. One of minimal, table, json (default: minimal)
   -s, --sort <sort>       Set the sort order for the coverage table. One of filename, +cov, -cov (default: filename)
   -d, --dependencies      Include dependencies in code coverage calculation. 
+  -r, --removetestfiles   Removes coverage details for all files containing "Tests" or "test" in their name
```

Comment:
Let's ditch the short-name and use the `@Flag` property builder's `inversion: .prefixNo` option with the name `"tests"` here so that the usage is:
```
  --tests/--no-tests      Determines whether test files are included in coverage calculation. (default: true)
```

I think when I created this project `ArgumentParser` didn't have that feature yet, but now that it does I wish that my `-d, --dependencies` were similarly the `--dependencies/--no-dependencies` flags.

Reply:
^ of course my suggestion also implies that the flag's default value is swapped (from a default of "don't remove test files" to a default of "do include test files")

Reply:
This looks cleaner. Making suggested changes.

Reply:
One suggestion here. I think making it `false` by default would be better. Because when a user is trying to parse coverage data, they would intend to get coverage details for well, only their product. Getting coverage for tests themselves is a rare IMHO.

Reply:
I've set the default behaviour to `--no-tests`. `includeTests` is false by default here. Let me know if this would be better, or if you would still prefer including tests themselves in total coverage by default.

Reply:
I think you're right. My goal was to make this change non-breaking, but we can go ahead and make the breaking change. I'll change the dependencies flag too since the next release will be breaking anyway.

Reply:
awesome. Will resolve this comment then. 👍🏽